monitoring/uss_qualifier/configurations/dev/faa/uft/local_message_signing.yaml

```diff
@@ -8,9 +8,7 @@ resources:
       specification:
         flight_planners:
         - participant_id: uss1
-          injection_base_url: http://host.docker.internal:8074/scdsc
-        - participant_id: uss2
-          injection_base_url: http://host.docker.internal:8074/scdsc
+          injection_base_url: http://host.docker.internal:10230/ussq
```

Comment:
Configurations in the `dev` folder are generally intended to enable contributors to the repository to exercise behavior using [uss_qualifier/run_locally.sh](https://github.com/interuss/dss/blob/master/monitoring/uss_qualifier/run_locally.sh).  If a developer needs to have something available at port 10230, I think we would need to document how they would need to prepare their system in order to use this configuration file (different than standard deployment [noted for run_locally.sh](https://github.com/interuss/dss/blob/d0144d857e4032064f3abbce52f6b9f0c2e76aed/monitoring/uss_qualifier/run_locally.sh#L16)).  Or, [the personal folder](https://github.com/interuss/dss/tree/master/monitoring/uss_qualifier/configurations/personal) can be used to store configurations that aren't intended to be run by other people.

Reply:
Yes, we will add a README in another PR to replace this url with anything that USS is using. I hope that's good enough.